colcon_alias/argument_parser/alias.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2022 Scott K Logan
+# Licensed under the Apache License, Version 2.0
+
+from colcon_alias.config import get_config
+from colcon_alias.verb.alias_invocation import AliasInvocationVerb
+from colcon_core.argument_parser import ArgumentParserDecorator
+from colcon_core.argument_parser import ArgumentParserDecoratorExtensionPoint
+from colcon_core.plugin_system import satisfies_version
+
+
+class AliasArgumentParserDecorator(ArgumentParserDecoratorExtensionPoint):
+    """Add command aliases verbs to the argument parser."""
+
+    # Because this parser adds subparsers, it needs to be added after
+    # other decorators like colcon-defaults that track them
+    PRIORITY = 50
```

Comment:
We actually want a lower number to achieve the desired effect. The two concerning `argument_parser` extensions are `defaults` and `mixin`, and both appear to inherit the default `PRIORITY` value of `100`, so we need only be less than that, at least as far as the "common" extensions are concerned.

I think that priority values can go negative, but I haven't actually tried it. In any case, we're 50 priorities away from the absolute requirements, so that should give future extensions room on either side of this one to fall.

Reply:
I'd guess I should clarify that I'm spacing things out because there are reasonable applications of the `argument_parser` extension which would want to be processed after this instance. I'm not aware of any released colcon packages that would need to do it right now, but it's a reasonable that there might be someday.

Reply:
:+1: